### PR TITLE
add OapTab in SparkUI

### PR DESCRIPTION
**Pull request by TigerSong**
## What changes were proposed in this pull request?

  * trigger OapTab in OapEnv
  * add a FiberCacheManagerPage
  * add UI related resources:js/html

## How was this patch tested?

image

### CODE REVIEW

**Comment by TigerSong** [comment]
image

File: src/main/resources/oap/static/oap.js [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+$(document).ajaxStart(function () {
+    $.blockUI({message: '<h3>Loading OAP Page...</h3>'});
+});
+
+function createTemplateURI(appId) {
+    var words = document.baseURI.split('/');
+    var ind = words.indexOf("proxy");
+    if (ind > 0) {
+        var baseURI = words.slice(0, ind + 1).join('/') + '/' + appId + '/static/oap/oap-template.html';
+        return baseURI;
```
Comment (LuciferYang) [inline src/main/resources/oap/static/oap.js]: Does baseURI need to assign to a var?

Comment (TigerSong) [reply src/main/resources/oap/static/oap.js]: Just copy from executorspage.js.

**Comment by carsonwang** [comment]
@TigerSong , we tested this patch but didn't see the OAP tab on UI. Have you tested the latest code in both Yarn and standalone? Or did we miss anything?

**Comment by TigerSong** [comment]
@carsonwang
OapTab is added in `OapEnv.init()`, which is the most suitable place in my mind, but seems only called when starts with `bin/spark-sql`.  I supposes that `OapEnv.init()` will be moved to a more suitable place in the future. 

Or maybe you can help me to find a more suitable place to add OapTab.

**Comment by carsonwang** [comment]
I had a try on the UI and REST API by running queries in "bin/spark-sql". The UI looks good so I am going to merge this. Thanks a lot @TigerSong for the work!

**Comment by LuciferYang** [comment]
Thx~ @carsonwang

**Pull request merged**